### PR TITLE
Test cases for complex and mutate stage

### PR DESCRIPTION
**Pull request by aggreen**

### CODE REVIEW

**Comment by thomashopf** [comment]
Quick question: do the tests in any way depend on #121 (which I am currently holding off looking at because of subclassing CouplingsModel for a ComplexCouplingsModel)?

**Comment by aggreen** [comment]
No - the tests are for the current monomer mutation stage only.

**Comment by sacdallago** [comment]
An effective way of seeing if tests are performed well is code coverage, a.k.a.: how many lines of code in a file are being executed by the tests.

I've spent some time researching, and I think a very neat app is this: https://github.com/codecov/example-python

I've already added EVcouplings [here](https://codecov.io/gh/debbiemarkslab/EVcouplings), if you could adapt the .travis file to [this](https://github.com/codecov/example-python/blob/master/.travis.yml) and test if the command works locally, that'd be awesome, and we'd have a nice report of how deep the tests go :)

File: evcouplings/mutate/protocol.py [hunk 1]
```diff
@@ -14,7 +14,9 @@
 from evcouplings.mutate.calculations import (
     single_mutant_matrix, predict_mutation_table
 )
-import evcouplings
+from evcouplings.visualize.mutations import (
+    plot_mutation_matrix, mutation_pymol_script
```
Comment (thomashopf) [inline evcouplings/mutate/protocol.py]: This is not blowing up with a circular import upon this change (which is why I had the ugly original solution I think)?

File: test/TestMutation.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
+"""
+Test cases for concatenation stage of EVCouplings complex pipline
```
Comment (thomashopf) [inline test/TestMutation.py]: mutation stage

File: evcouplings/mutate/protocol.py (hunk 1 above)
Comment (aggreen) [reply evcouplings/mutate/protocol.py]: Actually it was blowing up the way it was written, the only way to get rid of it was changing to what I have.... I can't explain it

**Comment by sacdallago** [comment]
This is a very pretty graph!!!!

**Pull request merged**